spec/features/user_adds_new_brewpub_spec.rb

```diff
@@ -0,0 +1,125 @@
+require "rails_helper"
```

Comment:
Generally, you should test all your validations in your model. In your feature, you should test 1 happy path and 1 UNIQUE sad path (happy: user submits information correctly, sad: user submits information incorrectly). It's just much easier to test the granular issues on the model level, and keeping your feature tests simple (Your sad path should test for a flash message, lack of creation, and rerendering of the form)